public/app/types/serviceaccount.ts

```diff
@@ -0,0 +1,96 @@
+import { OrgRole, Unit } from '.';
+import { SelectableValue } from '@grafana/data';
+
+export interface OrgServiceAccount {
+  avatarUrl: string;
+  email: string;
+  lastSeenAt: string;
+  lastSeenAtAge: string;
+  login: string;
+  name: string;
+  orgId: number;
+  role: OrgRole;
+  serviceAccountId: number;
+}
+
+export interface ServiceAccount {
+  id: number;
+  label: string;
+  avatarUrl: string;
+  login: string;
+  email: string;
+  name: string;
+  orgId?: number;
+}
+
+export interface ServiceAccountDTO {
+  id: number;
+  login: string;
+  email: string;
+  name: string;
+  isGrafanaAdmin: boolean;
+  isDisabled: boolean;
+  isAdmin?: boolean;
+  isExternal?: boolean;
+  updatedAt?: string;
+  authLabels?: string[];
+  theme?: string;
+  avatarUrl?: string;
+  orgId?: number;
+  lastSeenAtAge?: string;
+  licensedRole?: string;
```

Comment:
question: how does licensing work with service accounts? Are they counted the same as users?

Reply:
🤔 we should follow up on this after. I'l leave a comment and a issue to follow up